.github/workflows/dev_deploy.yml

```
@@ -0,0 +1,66 @@
+name: CI/CD Pipeline
+
+on:
+  push:
+    branches: [ develop ]  # develop 브랜치에 push가 일어날 때 실행
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3  # 저장소 코드 체크아웃
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)